src/main/webapp/dev/js/common/instructorEnroll.js

```diff
@@ -64,8 +64,8 @@ function ajaxDataToHandsontableData(studentsData, handsontableColHeader) {
 /**
  * Toggle the chevron image depending on the user's action.
  */
-function toggleChevronImage(panelCollapse, toggleChevron) {
-    if ($(panelCollapse[0]).hasClass('checked')) { // panel is shown
+function toggleChevronImage(isExpanded, toggleChevron) {
+    if (isExpanded) { // panel is shown
         $(toggleChevron).addClass('glyphicon-chevron-up').removeClass('glyphicon-chevron-down');
```

Comment:
Is the `$(...)` in `$(toggleChevron)` necessary? I think most likely not. Seems like you overused the jquery `$(...)` quite a bit huh. Usually, it's only used with a literal CSS selector to get the element you want once, and then it can be passed around and used directly

Reply:
Similar comment to https://github.com/TEAMMATES/teammates/pull/8884#discussion_r205968463.